radix-clis/assets/template/rust-toolchain.toml_template

```diff
@@ -0,0 +1,5 @@
+[toolchain]
+channel = "1.81.0"
```

Comment:
Nice :).

Can we update the instructions in `action.yml` to include updating this when we update the Rust version? (and remove the instructions which still refer to the `README.MD` but no longer need to?)